cmd/common.go

```diff
@@ -361,3 +361,40 @@ func Expand(s string) (string, error) {
 	}
 	return filepath.Abs(s)
 }
+
+func PushAccount(u string, accountjwt []byte) ([]byte, error) {
+	resp, err := http.Post(u, "application/text", bytes.NewReader(accountjwt))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		body, _ := ioutil.ReadAll(resp.Body)
+		m := ""
+		if body != nil {
+			var vr jwt.ValidationResults
```

Comment:
This is the code i just got rid of in my PR :-) but yours will just go to the err and print the string so should be ok

Reply:
I'll remove when I make the push cmd work the same.